src/exercise/dist/Main.js

```diff
@@ -0,0 +1,183 @@
+
+class Main {
+    constructor() {
+        // when sort button is clicked it sorts the tasks ascending/descending
+        this.asc_AZ = true;
+        this.asc_time = false;
+        this.itemClient = new ItemClient();
+    };
+
+
+    createTask(value, date){
+        const task_text = document.createElement("p");
+        task_text.innerText = value;
+        const task_elem = document.createElement("div");
+        task_elem.className = "tasks";
+        task_elem.onclick = () => alert("Task: \n" + task_text.innerText);
+        task_elem.appendChild(task_text);
+        task_elem.appendChild(this.taskTime(date));
+        return task_elem;
+    };
+
+    createDeleteBtn(task_section){
+        const delete_btn = document.createElement("button");
+        delete_btn.className = "deleteBtn";
+        delete_btn.onclick = () => {
+            task_section.classList.add("removed");
+            setTimeout(() => {
+                task_section.remove();
+                const task_date = task_section.querySelectorAll('p');
+                this.itemClient.deleteTask(task_date[0].innerText, task_date[1].id);
+            }, 800);
+            return;
+        }
+        const deleteIcon = document.createElement("i");
```

Comment:
Be consistent with variable naming - for JS it is recommended to use camelCase
so I suggest to rename the delete_btn element to deleteButton